jolt-core/src/zkvm/instruction_lookups/ra_virtual.rs

```diff
@@ -132,11 +128,9 @@ impl<F: JoltField> RASumCheck<F> {
         let ra_i_polys = Self::compute_ra_i_polys(trace, state_manager);
 
         // E_table[i] stores the evaluation of eq(r_cycle[i..], x), where i starts at 1.
-        let E_table = EqPolynomial::evals_cached_rev(r_cycle)
+        let E_table = EqPolynomial::evals_cached_rev(&r_cycle[1..])
             .into_iter()
-            .skip(1)
             .rev()
-            .skip(1)
             .collect::<Vec<_>>();
```

Comment:
Can we just do `EqPolynomial::evals()` here instead, and remove the `.rev()`, as well as enable parallelism? Or is caching required? 
The `EqPolynomial::evals_cached()` does evals only serially.

Reply:
Updated.